modules/nf-core/glimpse/ligate/meta.yml

```diff
@@ -0,0 +1,45 @@
+name: "glimpse_ligate"
+description: Concatenates imputation chunks in a single VCF/BCF file ligating phased information.
+keywords:
+  - ligate
+  - imputation
+tools:
+  - "glimpse":
+      description: "GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies."
+      homepage: "https://odelaneau.github.io/GLIMPSE"
+      documentation: "https://odelaneau.github.io/GLIMPSE/commands.html"
+      tool_dev_url: "https://github.com/odelaneau/GLIMPSE"
+      doi: "10.1038/s41588-020-00756-0"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - input:
+      type: file
+      description: Text file containing the full list of files to ligate (one file per line)
+      pattern: "*.txt"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+
+  - merged_bcf:
+      type: file
+      description: Output VCF/BCF file for the merged regions. Phased information (HS field) is updated accordingly for the full region.
```

Comment:
Is the output vcf AND bcf? Or just bcf?
If both, can you add output channels for either option?

Reply:
I think that both can be generated.
But I didn't tried yet.
I will test that !

Reply:
All possible outputs should be accounted for in the output channels, so we'd need to separate, optional outputs for vcf and bcf.
I suppose the output format can be set with some `ext.args` arguments?

Reply:
It doesn't seems to be the case. I think it is inferred from the --output extension.
There is more info [here](https://odelaneau.github.io/GLIMPSE/glimpse1/commands.html)

Reply:
Is a vcf output at all possible? I think vcf is more "standard" than bcf, but it has to fit in your flow of course.

Reply:
@matthdsm I tried both, and it seems effectively to automatically detect the output format based on the extension given as `--output`.
How do I control for that ?
Do I force the format or do I add another argument that check the format the user want ?

Reply:
_personally_ I'd default to vcf...
But since my opinion doesn't cover everyone's usecase, that doesn't matter. I think usually we check for the input format and output in the same format.

Reply:
Then you can add 2 _optional_ output channels for both the vcf and bcf.

Reply:
of course @nf-core/maintainers might have more opinions

Reply:
You can use the `task.ext.suffix` to define the extension. Maybe set `vcf` as default on this :)

Reply:
Is it better to use 2 different output channels or just one that get both as in samtools convert ?

Reply:
In this module it's one or the other, so I think you can keep it as one channel

Reply:
I would agree to just have one channel which takes the output, whichever file type it is, as it only generates one single file.